openstates/sd/people.py

```diff
@@ -0,0 +1,125 @@
+import re
+
+from pupa.scrape import Person, Scraper, Organization
+
+import lxml.html
+
+
+class SDLegislatorScraper(Scraper):
+    latest_only = True
+
+    def scrape(self, chambers=None):
+        url = 'http://www.sdlegislature.gov/Legislators/default.aspx' \
+              '?CurrentSession=True'
+        if chambers is None:
+            chambers = ['upper', 'lower']
+        for chamber in chambers:
+            if chamber == 'upper':
+                search = 'Senate Members'
+            else:
+                search = 'House Members'
+
+            page = self.get(url).text
+            page = lxml.html.fromstring(page)
+            page.make_links_absolute(url)
+
+            for link in page.xpath("//h4[text()='{}']/../div/a".format(search)):
+                name = link.text.strip()
+
+                yield from self.scrape_legislator(name, chamber,
+                                                  '{}&Cleaned=True'.format(
+                                                    link.attrib['href']))
+
+    def scrape_legislator(self, name, chamber, url):
+        page = self.get(url).text
+        page = lxml.html.fromstring(page)
+        page.make_links_absolute(url)
```

Comment:
What do you think about using `LXMLMixin` so we can call `self.lxmlize` and simplify some of this repeated code?